yt_dlp/extractor/nytimes.py

```diff
@@ -58,52 +48,27 @@ def get_file_size(file_size):
                 formats.extend(m3u8_fmts)
                 subtitles = self._merge_subtitles(subtitles, m3u8_subs)
             elif ext == 'mpd':
-                continue
-            #     formats.extend(self._extract_mpd_formats(
-            #         video_url, video_id, format_id or 'dash', fatal=False))
+                self.report_warning('Skipping DASH formats', video_id)
```

Comment:
if we make it non-fatal I don't see how it could hurt
```suggestion
                dash_fmts, dash_subs = self._extract_mpd_formats_and_subtitles(
                    video_url, video_id, fatal=False, mpd_id='dash')
                formats.extend(dash_fmts)
                self._merge_subtitles(dash_subs, target=subtitles)
```